src/java_bytecode/jar_file.cpp

```diff
@@ -48,8 +77,27 @@ void jar_filet::open(const std::string &filename)
         mz_zip_reader_get_filename(&zip, i, filename_char, filename_length);
       assert(filename_length==filename_len);
       std::string file_name(filename_char);
+
+      // remove WAR class prefix
+      const std::string war_class_prefix("WEB-INF/classes/");
```

Comment:
apparently, yes, cf. https://en.wikipedia.org/wiki/WAR_(file_format)
what's currently missing though, is loading of jars listed in `WEB-INF/lib`